### ISSUE TITLE
【EESS】New submissions for Wed,  9 Aug 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#493]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Proceedings of the first edition of the International Symposium on  Computational Sensing (ISCS23)
 - **Authors:** Thomas Feuillen, Amirafshar Moshtaghpour
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2308.04228
 - **Pdf link:** https://arxiv.org/pdf/2308.04228
 - **Abstract**
 The International Symposium on Computational Sensing (ISCS) brings together researchers from optical microscopy, electron microscopy, RADAR, astronomical imaging, biomedical imaging, remote sensing, and signal processing. With a particular focus on applications and demonstrators, the purpose of this symposium is to be a forum where researchers in computational sensing working in seemingly unrelated applications can learn, discover, and exchange on their new findings and challenges. This 3-day symposium in the heart of Europe features 6 keynotes speakers and is open to extended abstracts for scientific presentations and show-and-tell demonstrations.
### Data Augmentation-Based Unsupervised Domain Adaptation In Medical  Imaging
 - **Authors:** Sebastian Nørgaard Llambias, Mads Nielsen, Mostafa Mehdipour Ghazi
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2308.04395
 - **Pdf link:** https://arxiv.org/pdf/2308.04395
 - **Abstract**
 Deep learning-based models in medical imaging often struggle to generalize effectively to new scans due to data heterogeneity arising from differences in hardware, acquisition parameters, population, and artifacts. This limitation presents a significant challenge in adopting machine learning models for clinical practice. We propose an unsupervised method for robust domain adaptation in brain MRI segmentation by leveraging MRI-specific augmentation techniques. To evaluate the effectiveness of our method, we conduct extensive experiments across diverse datasets, modalities, and segmentation tasks, comparing against the state-of-the-art methods. The results show that our proposed approach achieves high accuracy, exhibits broad applicability, and showcases remarkable robustness against domain shift in various tasks, surpassing the state-of-the-art performance in the majority of cases.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Advancing Natural-Language Based Audio Retrieval with PaSST and Large  Audio-Caption Data Sets
 - **Authors:** Paul Primus, Khaled Koutini, Gerhard Widmer
 - **Subjects:** Audio and Speech Processing (eess.AS); Information Retrieval (cs.IR); Machine Learning (cs.LG); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2308.04258
 - **Pdf link:** https://arxiv.org/pdf/2308.04258
 - **Abstract**
 This work presents a text-to-audio-retrieval system based on pre-trained text and spectrogram transformers. Our method projects recordings and textual descriptions into a shared audio-caption space in which related examples from different modalities are close. Through a systematic analysis, we examine how each component of the system influences retrieval performance. As a result, we identify two key components that play a crucial role in driving performance: the self-attention-based audio encoder for audio embedding and the utilization of additional human-generated and synthetic data sets during pre-training. We further experimented with augmenting ClothoV2 captions with available keywords to increase their variety; however, this only led to marginal improvements. Our system ranked first in the 2023's DCASE Challenge, and it outperforms the current state of the art on the ClothoV2 benchmark by 5.6 pp. mAP@10.
## Keyword: video retrieval
There is no result 
### Reconfigurable and Static EM Skins on Vehicles for Localization
 - **Authors:** Dario Tagliaferri, Marouan Mizmizi, Giacomo Oliveri, Umberto Spagnolini, Andrea Massa
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2308.04319
 - **Pdf link:** https://arxiv.org/pdf/2308.04319
 - **Abstract**
 Electromagnetic skins (EMSs) have been recently considered as a booster for wireless sensing, but their usage on mobile targets is relatively novel and could be of interest when the target reflectivity can/must be increased to improve its detection or the estimation of parameters. In particular, when illuminated by a wide-bandwidth signal (e.g., from a radar operating at millimeter waves), vehicles behave like \textit{extended targets}, since multiple parts of the vehicle's body effectively contribute to the back-scattering. Moreover, in some cases perspective deformations challenge the correct localization of the vehicle. To address these issues, we propose lodging EMSs on vehicles' roof to act as high-reflectivity planar retro-reflectors toward the sensing terminal. The advantage is twofold: \textit{(i)} by introducing a compact high-reflectivity structure on the target, we make vehicles behave like \textit{point targets}, avoiding perspective deformations and related ranging biases and \textit{(ii)} we increase the reflectivity the vehicle, improving localization performance. We detail the EMS design from the system-level to the full-wave-level considering both reconfigurable intelligent surfaces (RIS) and cost-effective static passive electromagnetic skins (SP-EMSs). Localization performance of the EMS-aided sensing system is also assessed by Cram\'er-Rao bound analysis in both narrowband and spatially wideband operating conditions.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
### Nest-DGIL: Nesterov-optimized Deep Geometric Incremental Learning for CS  Image Reconstruction
 - **Authors:** Xiaohong Fan, Yin Yang, Ke Chen, Yujie Feng, Jianping Zhang
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2308.03807
 - **Pdf link:** https://arxiv.org/pdf/2308.03807
 - **Abstract**
 Proximal gradient-based optimization is one of the most common strategies for solving image inverse problems as well as easy to implement. However, these techniques often generate heavy artifacts in image reconstruction. One of the most popular refinement methods is to fine-tune the regularization parameter to alleviate such artifacts, but it may not always be sufficient or applicable due to increased computational costs. In this work, we propose a deep geometric incremental learning framework based on second Nesterov proximal gradient optimization. The proposed end-to-end network not only has the powerful learning ability for high/low frequency image features,but also can theoretically guarantee that geometric texture details will be reconstructed from preliminary linear reconstruction.Furthermore, it can avoid the risk of intermediate reconstruction results falling outside the geometric decomposition domains and achieve fast convergence. Our reconstruction framework is decomposed into four modules including general linear reconstruction, cascade geometric incremental restoration, Nesterov acceleration and post-processing. In the image restoration step,a cascade geometric incremental learning module is designed to compensate for the missing texture information from different geometric spectral decomposition domains. Inspired by overlap-tile strategy, we also develop a post-processing module to remove the block-effect in patch-wise-based natural image reconstruction. All parameters in the proposed model are learnable,an adaptive initialization technique of physical-parameters is also employed to make model flexibility and ensure converging smoothly. We compare the reconstruction performance of the proposed method with existing state-of-the-art methods to demonstrate its superiority. Our source codes are available at https://github.com/fanxiaohong/Nest-DGIL.
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### A Projection-Based Approach for Distributed Energy Resources Aggregation
 - **Authors:** Yiran Wang, Haiwang Zhong, Guangchun Ruan
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2308.04007
 - **Pdf link:** https://arxiv.org/pdf/2308.04007
 - **Abstract**
 Aggregating distributed energy resources (DERs) is of great significance to improve the overall operational efficiency of smart grid. The aggregation model needs to consider various factors such as network constraints, operational constraints, and economic characteristics of the DERs. This paper constructs a multi-slot DER aggregation model that considers the above factors using feasible region projection approach, which achieved the protection of DERs data information and the elimination of internal variables. A system economic dispatch (ED) model is established for the operators to make full use of the DER clusters. We calculate the feasible regions with temporal coupling by extending the Progressive Vertex Enumeration (PVE) algorithm to high dimension by the Quickhull algorithm. Finally, an IEEE 39-bus distribution network is simulated with DERs to verify the effectiveness of the proposed model. Results show that the two-step ED derives the same results as the centralized ED.
### SDLFormer: A Sparse and Dense Locality-enhanced Transformer for  Accelerated MR Image Reconstruction
 - **Authors:** Rahul G.S., Sriprabha Ramnarayanan, Mohammad Al Fahim, Keerthi Ram, Preejith S.P, Mohanasankar Sivaprakasam
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2308.04262
 - **Pdf link:** https://arxiv.org/pdf/2308.04262
 - **Abstract**
 Transformers have emerged as viable alternatives to convolutional neural networks owing to their ability to learn non-local region relationships in the spatial domain. The self-attention mechanism of the transformer enables transformers to capture long-range dependencies in the images, which might be desirable for accelerated MRI image reconstruction as the effect of undersampling is non-local in the image domain. Despite its computational efficiency, the window-based transformers suffer from restricted receptive fields as the dependencies are limited to within the scope of the image windows. We propose a window-based transformer network that integrates dilated attention mechanism and convolution for accelerated MRI image reconstruction. The proposed network consists of dilated and dense neighborhood attention transformers to enhance the distant neighborhood pixel relationship and introduce depth-wise convolutions within the transformer module to learn low-level translation invariant features for accelerated MRI image reconstruction. The proposed model is trained in a self-supervised manner. We perform extensive experiments for multi-coil MRI acceleration for coronal PD, coronal PDFS and axial T2 contrasts with 4x and 5x under-sampling in self-supervised learning based on k-space splitting. We compare our method against other reconstruction architectures and the parallel domain self-supervised learning baseline. Results show that the proposed model exhibits improvement margins of (i) around 1.40 dB in PSNR and around 0.028 in SSIM on average over other architectures (ii) around 1.44 dB in PSNR and around 0.029 in SSIM over parallel domain self-supervised learning. The code is available at https://github.com/rahul-gs-16/sdlformer.git
### Achieving Covert Communication With A Probabilistic Jamming Strategy
 - **Authors:** Xun Chen, Fujun Gao, Min Qiu, Jia Zhang, Feng Shu, Shihao Yan
 - **Subjects:** Signal Processing (eess.SP); Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2308.04278
 - **Pdf link:** https://arxiv.org/pdf/2308.04278
 - **Abstract**
 In this work, we consider a covert communication scenario, where a transmitter Alice communicates to a receiver Bob with the aid of a probabilistic and uninformed jammer against an adversary warden's detection. The transmission status and power of the jammer are random and follow some priori probabilities. We first analyze the warden's detection performance as a function of the jammer's transmission probability, transmit power distribution, and Alice's transmit power. We then maximize the covert throughput from Alice to Bob subject to a covertness constraint, by designing the covert communication strategies from three different perspectives: Alice's perspective, the jammer's perspective, and the global perspective. Our analysis reveals that the minimum jamming power should not always be zero in the probabilistic jamming strategy, which is different from that in the continuous jamming strategy presented in the literature. In addition, we prove that the minimum jamming power should be the same as Alice's covert transmit power, depending on the covertness and average jamming power constraints. Furthermore, our results show that the probabilistic jamming can outperform the continuous jamming in terms of achieving a higher covert throughput under the same covertness and average jamming power constraints.
